src/http/client/client.cr

```diff
@@ -167,6 +167,36 @@ class HTTP::Client
     self.connect_timeout = connect_timeout.total_seconds
   end
 
+  # Set the number of seconds to wait when resolving name, before raising an `IO::Timeout`.
+  #
+  # ```
+  # client = HTTP::Client.new("grtv.ml")
+  # client.dns_timeout = 1.5
+  # begin
+  #   response = client.get("/")
+  # rescue IO::Timeout
+  #   puts "Timeout!"
+  # end
+  # ```
+  def dns_timeout=(dns_timeout : Number)
+    @dns_timeout = dns_timeout.to_f
+  end
+
+  # Set the open timeout with a `Time::Span` to wait when resolving name, before raising an `IO::Timeout`.
```

Comment:
let's keep more consistent wording with the previous one here. Since DNS is usually UDP, open timeout seems wrong anyway.